### DISCUSSION TITLE
v1.0-Alpha-01

### DISCUSSION THREAD

**Discussion by DevVigilante** [Innovative-CST/Android-Code-Editor#12] [Announcements]
📢 Release Notes - 1.0-Alpha-01🚀

🎉 Exciting Features:

1. Mixed Editor Widget: The app offers a powerful Mixed Editor Widget, providing multiple editor options within a single interface.
   - Sora Editor
   - Ace Editor
You can choose any editor for different coding experience.

We have also added zooming feature for Ace editor that was not supported by library.

Dynamic Colors: Enjoy dynamic colors in the app to enhance your coding environment.

Theme Palette (Only available if Dynamic colors not available in device): If dynamic colors are not supported on your device, you can still customize the app's theme using the theme palette.

Save (Auto) and Edit Files: The app supports automatic saving and editing of files, ensuring your changes are always preserved.

Project Creation: Easily create new coding projects within the app, helping you stay organized and manage your files efficiently.

File Tree: Access and navigate through your project's file structure with the new file tree feature, making it convenient to locate and work with specific files.

Inbuilt Browser for HTML File Preview: Preview HTML files in real-time using the inbuilt browser feature, allowing you to see how your web content looks without leaving the app.

Markdown File Preview: View and preview Markdown files directly within the app, making it convenient to review and validate the formatting of your Markdown documents.

Supported Language Modes: The app supports several language modes, enabling syntax highlighting and code suggestions for popular programming languages.
   - Java
   - JavaScript
   - HTML
   - CSS
   - XML
   - Markdown

We hope you enjoy using these new features! Your feedback is valuable to us as we continue to improve the app. Stay tuned for more updates and enhancements in future releases. Happy coding! ✨👩‍💻👨‍💻✨

<hr /><em>This discussion was created from the release <a href='https://github.com/TS-Code-Editor/Android-Code-Editor/releases/tag/v1.0-Alpha-01'>v1.0-Alpha-01</a>.</em>

**Comment by CyberGlitch01** [comment]
@DevKumar00001 You have not mentioned some minor features.

**Reply by DevVigilante** [reply]
It is first release so there was nothing to compare with.
